From: SeongJae Park <sj@kernel.org>
Date: Wed, 7 Jan 2026 17:42:43 -0800
Subject: [PATCH] ==== selftest fixups ====

selftests/damon: make leak detection and wss estimation reliable

Two DAMON selftets, namely 'sysfs_memcg_leak' and
'sysfs_update_schemes_tried_regions_wss_estimation' frequently shows
intermittent failures.  Improve their reliability.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus__x8brb_m | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus__x8brb_m

diff --git a/hkml_cv_bogus/hkml_cv_bogus__x8brb_m b/hkml_cv_bogus/hkml_cv_bogus__x8brb_m
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.47.3

